src/client/src/App.tsx

```diff
@@ -80,6 +81,7 @@ interface IDispatchProps {
   updateTemplateGenStatusMessage: (status: string) => any;
   updateTemplateGenStatus: (isGenerated: IServiceStatus) => any;
   getVersionsData: (versions: IVersions) => any;
+  updateDependencyInfo: (dependencyInfo: any) => any;
```

Comment:
Just noticed this as well, can you also add the IDependencyInfo to this in this PR or your refactoring PR? @jsondoo

Reply:
Hey @jsondoo think you forgot this one

Reply:
oh man, whoops, thanks @SahilTara